doc/src/PYIODE.m

```diff
@@ -7,54 +7,33 @@
 
 &TI Introduction
 �����������������
-The python module for IODE implements part of the IODE API. 
-The aim is to implement a python layer for all IODE functionalities so that IODE can be driven directly from a python session.
-
-&TI Python version & iode.pyd
-�����������������
-Each version of python has its own IODE module ~ciode.pyd~C. They are stored in 
-different subdirectories of the IODE main directory (usually ~cc:\iode~C). For example: ~c.\py39, .\py310~C...
-By default, the ~ciode.pyd~C module for python 3.9 is thus ~cc:/iode/py39/iode.pyd~C. For python 3.10: ~cc:/iode/py310/python.pyd~C.
+The python package for IODE implements part of the IODE API. 
+The aim is to implement a python layer for all IODE functionalities so that 
+IODE can be driven directly from a Python console or script.
 
 
-&TI Memory management
+&TI Installation
 �����������������
-During a python session, IODE data are grouped in workspaces, one by object type (variables, scalars, equations...) 
-and stored in IODE memory using a proprietary memory management system called IODE-SWAP. IODE functions operate ~binside~B that memory.
- 
-Exchanges between IODE objects (in IODE-SWAP) and python objects (larray, numpy or pandas objects in python memory) 
-are made possible via interface functions described below like ~cget_eqs()~C, ~cset_eqs()~C or ~clarray_to_ws()~C.
+You must first download an installation file with the extension whl (called a Python wheel file).
```

Comment:
@jmpplan In the PYIODE.m file, I don't know to add a link to the download page of IODE. 
Could you tell me how to do that please ?